src/js/popup.js

```diff
@@ -117,6 +121,15 @@ async function showRelayPanel(tipPanelToShow) {
     emailsBlockedValEl.textContent = emailsBlockedVal;
     emailsForwardedValEl.textContent = emailsForwardedVal;
 
+    //Show premium panel state
+    if (premium && premiumEnabledString === "True"){
+      premiumPanelWrapper.classList.remove("is-hidden");
+      premiumPanelWrapper.querySelectorAll(".is-hidden").forEach(premiumFeature => 
+        premiumFeature.classList.remove("is-hidden") 
+        );
+      //Toggle register domain or education module
+      checkUserSubdomain(premiumSubdomainSet);
```

Comment:
After checking if user is premium, this line goes on to check which module to show depending on whether a subdomain is set or not.

Reply:
Since all the premium UI blocks are set to visible when they're premium, depending on whether or not a user has a subdomain, this function will determine which module to display by hiding the opposing one (register domain CTA vs educational module).